src/core/broadcaster.ts

```diff
@@ -58,7 +59,10 @@ export module Core {
      * @returns {Broadcaster} The calling Broadcaster
      */
     public broadcast(...args: any[]) {
-      this._key2callback.values().forEach((callback) => callback(this._listenable, args));
+      args.unshift(this._listenable);
```

Comment:
Oh I thought the input was an array being inputted from the user, not a bunch of separate args.  Never mind.